testcases/kernel/syscalls/lgetxattr/lgetxattr02.c

```diff
@@ -83,12 +93,19 @@ static void setup(void)
 	}
 }
 
+void cleanup(void)
+{
+        remove("testfile");
+        remove("symlink");
+}
+
 static struct tst_test test = {
-	.needs_tmpdir = 1,
+	// Removed directory creation and mounting tmp filesystem
```

Comment:
@vamsikrishna935 this explanation is too brief. Please explain how it is using root file system if you remove .needs+tmpdir = 1 . Also shall we put this bach when a bug get fixed, which issue. May be you should put TODO comment here. Also please don't remove needs_tmpdir and setup just comment out with necessary explanation why? also add comment shall we enable them back, when?

Reply:
@hukoyu Updated description. Issue 236 which is related tmp directory is fixed but still the issue persists. I think we need create a separate bug. Please let me know your thoughts.

Reply:
@hukoyu  Created an issue, lsds/sgx-lkl#736
Sean comments in issue 736:
prp has suggested we may not want to support this attribute on /tmp. Needs a triage discussion to determine if this is a by-design or P2/P3

I think we don't need a TODO here, it needs to be implemented in root filesystem only.